src/test/java/core/basesyntax/service/operations/SupplyOperationTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.exceptions.InvalidFruitException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class SupplyOperationTest {
+    private FruitTransaction transaction;
+    private final OperationHandler operation = new SupplyOperation();
+
+    @BeforeEach
+    void beforeAll() {
+        Storage.clear();
+        Storage.put(new Fruit("apple"), 10);

```

Comment:
In the 'beforeAll' method, you are adding an 'apple' with quantity 10 to the storage. This might affect the results of your other test cases if they are also using the 'Storage' class. You should clear the storage in the @AfterEach method as well to ensure each test case is independent.

Reply:
ok, done